### ISSUE TITLE
 java.lang.IllegalStateException: Native module Vibration tried to override com.facebook.react.modules.vibration.VibrationModule. Check the getPackages() method in MainApplication.java, it might be that module is being created twice. If this was your intention, set canOverrideExistingModule=true. This error may also be present if the package is present only once in getPackages() but is also automatically added later during build time by autolinking. Try removing the existing entry and rebuild.  

### ISSUE THREAD

**Issue by srouth1994** [itinance/react-native-fs#1117]
I have the same issue during the installation of react-native-video package.
![Screenshot_1657522497](https://user-images.githubusercontent.com/53264267/178212778-f21d4a99-12b6-429b-b3e2-ca21fcc71b78.png)

**Comment by kongshu612** [comment]
As the error said, just remove the code from MainApplicaiton.java.  Seems, it is already done by react native link in RN > 0.68.2 . we don't need add these code manually
```
import com.rnfs.RNFSPackage; // <------- add package

public class MainApplication extends Application implements ReactApplication {
   // ...
    @Override
    protected List<ReactPackage> getPackages() {
      return Arrays.<ReactPackage>asList(
        new MainReactPackage(), // <---- add comma
        new RNFSPackage() // <---------- add package
      );
    }
```

**Comment by flexbox** [comment]
Hey @srouth1994 

If your problem is solved can you please close the issue? 
It sparks joy for maintainers ✨
